widget/entry_internal_test.go

```diff
@@ -93,6 +93,29 @@ func TestEntry_DragSelect(t *testing.T) {
 	assert.Equal(t, "r the laz", entry.SelectedText())
 }
 
+func TestEntry_DragSelectWithScroll(t *testing.T) {
+	entry := NewEntry()
+	entry.SetText("The quick brown fox jumped over and over the lazy dog.")
+	entry.Resize(entry.MinSize())
+
+	// get position after the letter 'a'.
+	ev1 := getClickPosition("The quick brown fox jumped over and over the la", 0)
+	// get position after the letter 'u'
+	ev2 := getClickPosition("The qu", 0)
+
+	// mouse down and drag from 'a' to 'i'
+	me := &desktop.MouseEvent{PointEvent: *ev1, Button: desktop.MouseButtonPrimary}
+	entry.MouseDown(me)
+	for ; ev1.Position.X > ev2.Position.X; ev1.Position.X-- {
```

Comment:
You don't have to drag all increments - just mouse down at `ev1`, drag to `ev2` and mouse up at `ev2`.

Reply:
Got it. Done.